frontend/public/models/yaml-templates.ts

```diff
@@ -7,6 +7,10 @@ import { YAMLTemplate } from '@console/plugin-sdk';
 /**
  * Sample YAML manifests for some of the statically-defined Kubernetes models.
  */
+
+const containerImageSrc = 'registry.redhat.io/rhel8/support-tools';
```

Comment:
We probably want to reference the local tools image stream. @wking @jwforres Can you help us determine what is right?

For the node terminal, we try to get the image stream, but fall back to `registry.redhat.io/rhel8/support-tools` if we can't. That's tricky here since these samples are static today. It would be a fair amount of rework if we had to wait for an API request to resolve before we can get the sample.